web/packages/teleport/src/config.ts

```diff
@@ -1686,6 +1690,23 @@ const cfg = {
     }
   },
 
+  getWorkloadIdentityUrl(
+    req: {
+      action: 'list';
+    } & { clusterId?: string }
+  ) {
+    const { clusterId = cfg.proxyCluster } = req;
+    switch (req.action) {
+      case 'list':
+        return generatePath(cfg.api.workloadIdentity.list, {
+          clusterId,
+        });
+      default:
+        req.action satisfies never;
+        return '';
+    }
```

Comment:
i know this satisfies never and wont be hit but should we throw an error instead of returning empty string?